torch/lib/TH/THGeneral.h.in

```diff
@@ -42,8 +42,14 @@
 typedef void (*THErrorHandlerFunction)(const char *msg, void *data);
 typedef void (*THArgErrorHandlerFunction)(int argNumber, const char *msg, void *data);
 
+#define TH_DESC_BUFF_LEN 64
```

Comment:
Given that printing the size for a 4D tensor with 3 digits sizes in each dimension takes only 24, don't we want to lower this to 32 ?